go/vt/worker/multi_split_diff.go

```diff
@@ -407,7 +407,7 @@ func (msdw *MultiSplitDiffWorker) stopReplicationOnSourceTabletAt(ctx context.Co
 		msdw.wr.Logger().Infof("stopping Replication %v at a minimum of %v", msdw.sourceAlias, vreplicationPos)
 
 		shortCtx, cancel = context.WithTimeout(ctx, *remoteActionsTimeout)
-		msdw.wr.TabletManagerClient().StartReplication(shortCtx, sourceTablet.Tablet)
+		msdw.wr.StartReplication(shortCtx, sourceTablet.Tablet)
```

Comment:
This is brilliant. MultiSplitDiffWorker is created using a vtctld command which means we have the durability policy and wrangler can enforce it. 💯

Reply:
It is actually created from vtworker, but we have added durability policy to that too! So everything works now!